FTL.h

```diff
@@ -252,3 +257,5 @@ long int lastdbindex;
 bool travis;
 bool DBdeleteoldqueries;
 bool rereadgravity;
+char * clientip[MAXCONNS];
```

Comment:
We might even want to get rid of all of this as we aren't implementing an authentication directly in `FTL` now.